tetris.py

```diff
@@ -175,6 +176,15 @@ def draw_board(canvas):
         x_gap += square_size
 
 
+def new_game():
+    small_board = Board(topbar_canvas, None)
+    main_board = Board(game_canvas, small_board)
+    game = Game(main_board, small_board, topbar_score, topbar_time)
+    game.move_current_block_down()
+    small_board.draw_block(game.get_upcoming_block_shape(), game.upcoming_block_letter, game.landed_blocks)
```

Comment:
landed blocks not needed for small board:
```suggestion
    small_board.draw_block(game.get_upcoming_block_shape(), game.upcoming_block_letter)
```